### DISCUSSION TITLE
screenshot to vault

### DISCUSSION THREAD

**Discussion by louis030195** [coddingtonbear/obsidian-web#20] [Ideas]
On Mac I use SHIRT+CMD+4 to capture a part of the screen, It would be great to pop up a dialog afterward that let me choose where to send it in the vault, with additional text, somehow integrated with the templates

**Comment by ceciltech** [comment]
If you ~~have already installed the REST API plugin~~ install [Obsidian Advanced URI](https://github.com/Vinzent03/obsidian-advanced-uri) then you should be able to use the mac shortcuts app to do this and it has the advantage of working from any app not just chrome.
